src/rust/src/x509/verify.rs

```diff
@@ -101,11 +104,50 @@ impl PyServerVerifier {
 
     fn verify<'p>(
         &self,
-        _py: pyo3::Python<'p>,
-        _leaf: &PyCertificate,
-        _intermediates: &'p pyo3::types::PyList,
+        py: pyo3::Python<'p>,
+        leaf: &PyCertificate,
+        intermediates: &'p pyo3::types::PyList,
```

Comment:
Pretty sure you can just specify `Vec<PyRef<...>>` here?

Reply:
ef9061f3e